Crawler/Analyzers/UtilAnalyzers/IPunctuationAnalyzer.cs

```diff
@@ -14,5 +15,7 @@ public interface IPunctuationAnalyzer
 		int CalculateWordsCountDecile(int decile, List<Token> tokens);
 
 		int CountCharacter(char chr, List<Token> tokens);
+
+		double CalculatePassiveVoiceSentencesPercentage(List<PosTagToken> posTagTokens);
```

Comment:
Maybe it should be in another analyzer, sentence is determined by punctuation but I don't see the connection

Reply:
Hmm, I will consider it.